config/locales/en.yml

```diff
@@ -247,7 +247,7 @@ en:
       postcode_format: "Enter a real postcode"
       negative_date: "Enter a real %{field}"
       date_not_a_number: "Enter %{field} as a number"
-      telephone_number_format: "Enter a telephone number, like 01632 960 001, 07700 900 982 or +44 0808 157 0192"
+      telephone_number_format: "Enter a telephone number, like 02033 445 566, 07711330382 or +44 0808 157 0192"
```

Comment:
The choice of invalid telephone numbers here is deliberate, IIRC.  The numbers are from https://www.ofcom.org.uk/phones-telecoms-and-internet/information-for-industry/numbering/numbers-for-drama and are used so they definitely are not real number.

Reply:
That's interesting, thanks Bruce.

It looks like the gem or the underlying library is rejecting numbers from Ofcom's list, so I'll use Ofcom's numbers in the error text, but real (made up) numbers in the tests, if that's OK?

```ruby
TelephoneNumber.parse("07700900999", :gb).valid? # => false
```

As an aside, we should make sure that we don't send text messages in unit / smoke tests! :)

Reply:
That sounds like a good plan :+1: